modules/installation-creating-aws-worker.adoc

```diff
@@ -180,9 +180,11 @@ the CloudFormation template for the security group and roles.
 section of this topic and save it as a YAML file on your computer. This template
 describes the networking objects and load balancers that your cluster requires.
 
-. If you specified an `m5` instance type as the value for `WorkerInstanceType`,
-add that instance type to the `WorkerInstanceType.AllowedValues` parameter
-in the CloudFormation template.
+. If you specified an `m5` instance type as the value for `WorkerInstanceType`, add that instance type to the `WorkerInstanceType.AllowedValues` parameter in the CloudFormation template.
```

Comment:
If I am reading this correctly, you forgot to delete this step after adding it as optional.